tower-http/CHANGELOG.md

```diff
@@ -33,24 +53,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `Box<dyn std::error::Error + Send + Sync>`. This makes them usable if
   the body they're wrapping uses `Box<dyn std::error::Error + Send + Sync>` as
   its error type which they previously weren't ([#166]) (BREAKING)
-- **compression, decompression**: Remove the `compression` and `decompression` feature. They were unnecessary
-  and `compression-full`/`decompression-full` can be used to get full
-  compression/decompression support. For more granular control `[compression|decompression]-gzip`,
-  `[compression|decompression]-br` and `[compression|decompression]-deflate` may
-  be used instead ([#170]) (BREAKING)
 - **fs**: Changed response body type of `ServeDir` and `ServeFile` to
```

Comment:
it looks like most of the other changelog entries use present tense, but this one is past tense:
```suggestion
- **fs**: Change response body type of `ServeDir` and `ServeFile` to
```